crates/nebula-backbone/src/server/router/path/mod.rs

```diff
@@ -29,6 +29,7 @@ pub(crate) fn router(application: Arc<Application>) -> axum::Router {
             "/workspaces/:workspace_name/paths/*path",
             get(handle_get_path).delete(handle_delete_path).patch(handle_patch_path),
         )
+        .route_layer(middleware::from_fn(check_member_role))
```

Comment:
I think restricting HTTP methods like PATCH, DELETE, and POST to admin users only